dvc/commands/plots.py

```diff
@@ -70,8 +70,13 @@ def run(self):
                     "visualization file will not be created."
                 )
 
+            out: str = self.args.out or "dvc_plots"
+
+            renderers_out = (
+                out if self.args.json else os.path.join(out, "static")
+            )
```

Comment:
Seems like during extracting of the renderers we dropped saving images to dir by default. I think it's more convenient regarding parsing and potentially obtaining the images from produced HTML if we keep that behavior. @daavoo Was there a particular reason for this change? If there is, we can roll this back and change the tests accordingly.

Reply:
No reason. I assumed that we saved to `dir` by default because there was no support for embedding `base64`. 

After I added support for `base64`, I assumed that if someone wanted to use the "save to dir" behavior, they will explicitly use the `out` argument

Reply:
Got it, my reasoning for creating `dir` structure was to make HTML easier to read and have direct access to images in `static` folder.

Reply:
One thing we overlook here is that users might want to use the embedded behavior and after these changes we don't have a way to opt-out from the `dir` creation.

Reply:
One thing we overlook here is that users might want to use the embedded behavior and after these changes we don't have a way to opt-out from the `dir` creation.